manifest/xyz.jvyden/QuickPick/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "QuickPick",
+	"id": "xyz.jvyden.QuickPick",
+	"description": "A ResoniteModLoader mod that lets you quickly pick ProtoFlux nodes and components in desktop mode.",
+	"category": "Controls",
+	"sourceLocation": "https://github.com/jvyden/QuickPick",
+	"versions": {
+		"1.0.0": {
+			"releaseUrl": "https://github.com/jvyden/QuickPick/releases/tag/v1.0.0",
+			"artifacts": [{
+				"url": "https://github.com/jvyden/QuickPick/releases/download/v1.0.0/QuickPick.dll",
```

Comment:
Looks good.

I'd recommend setting the mod metadata directly for the version either with a constant (You can set the assembly data with it instead of in the csproj to still change it in 1 place.) https://github.com/resonite-modding-group/resonite-mod-manifest/wiki/Submission-Guidelines#mod-metadata or with a string normally.